specs/Autofill.md

```diff
@@ -0,0 +1,104 @@
+# Background
+The WebView2 team has been asked for an API to allow end developers to set the autofill preferences.  We are exposing two of the autofill preferences that will allow enabling and disabling general and password autofill.  General autofill includes things like email addresses, shipping addresses, phone numbers, and names.  Password autofill includes things like usernames and passwords for login. 
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+
+# Description
+
+Autofill has three components
+* Auto-populate - Populate the corresponding form fields automatically on page load.
+* Suggest - When the user clicks on the form field, drop down suggestions of previously saved forms will be displayed.
+* Populate - When clicking on one of the suggestions, the form data will populate the respective fields.
+
+
```

Comment:
I feel like we're missing more text here. Description should describe the API but gives a bit of info about what autofill consists of. Maybe additional info after to say what the two APIs do regarding the three parts of autofill?